R/pkg/R/generics.R

```diff
@@ -1046,8 +1084,8 @@ setGeneric("ntile", function(x) { standardGeneric("ntile") })
 #' @export
 setGeneric("n_distinct", function(x, ...) { standardGeneric("n_distinct") })
 
-#' @rdname percent_rank
-#' @export
+# @rdname percent_rank
```

Comment:
This is perhaps not the best way, but I did this because otherwise the generated doc will show `percent_rank(x)`. Then we have to document `x`, which could be confusing to the user since this argument should be left missing in actual function call.